Assets/UX/Scripts/ARUXReasonsManager.cs

```diff
@@ -85,11 +88,17 @@ public Sprite noneSprite
         set => m_NoneSprite = value;
     }
 
+    [SerializeField]
+    LocalizationManager m_LocalizationManager;
+
+    [SerializeField]
+    bool m_LocalizeText = true;
+
     NotTrackingReason m_CurrentReason;
     bool m_SessionTracking;
 
-    const string k_InitRelocalText = "Initializing augmented reality";
-    const string k_MotionText = "Try moving at a slower pace";
+    const string k_InitRelocalText = "Initializing augmented reality.";
```

Comment:
Probably not big deal, but since these are constants they will be always using memory even if the user chooses another language. I suggest treating the initial English strings as if they were other localized version